src/actions/create.js

```diff
@@ -37,10 +37,13 @@ const askAppName = () =>
     appName => validateAppName(appName),
   ])
 
-const askAppId = () =>
+const askAppId = appName =>
   sequence([
     () =>
-      ask('What is the App identifier? (reverse-DNS format)', 'com.metrological.app.myawesomeapp'),
+      ask(
+        'What is the App identifier? (reverse-DNS format)',
+        `com.metrological.app.${appName.replace(/ +/g, '').toLowerCase()}`
```

Comment:
My suggestion is to change the expression from `/ +/g` to `/\s+/g` first for readability but also to capture all whitespace characters.